src/components/JustNotSorry.js

```diff
@@ -41,14 +41,24 @@ class JustNotSorry extends Component {
     this.setState({ parentNode: {}, warnings: [] });
   }
 
-  updateWarnings(elem, patterns) {
-    const newWarnings = findRanges(elem, patterns);
-    this.setState({ parentNode: elem.parentNode, warnings: newWarnings });
+  updateWarnings(email, patterns) {
+    const newWarnings =
+      email.children.size > 0
```

Comment:
It seems like this should be `email.children.length`.

Reply:
I blame Java.... You are correct.